src/displayapp/screens/WatchFaceDigital.cpp

```diff
@@ -106,6 +110,19 @@ void WatchFaceDigital::Refresh() {
   if (batteryPercentRemaining.IsUpdated()) {
     auto batteryPercent = batteryPercentRemaining.Get();
     lv_label_set_text(batteryIcon, BatteryIcon::GetBatteryIcon(batteryPercent));
+
+    if (settingsController.GetBatteryColorStatus() == Controllers::Settings::BatteryColor::ON) {
+      lv_obj_set_style_local_text_color(batteryIcon, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, BatteryIcon::GetBatteryColor(batteryPercent));
+    } else {
+      lv_obj_set_style_local_text_color(batteryIcon, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, BatteryIcon::GetDefaultBatteryColor());
+    }
+    
+    if (settingsController.GetBatteryPercentageStatus() == Controllers::Settings::BatteryPercentage::ON) {
+      lv_label_set_text_fmt(batteryValue, "%lu", batteryPercent);
```

Comment:
Would it be better to print percentage sign?
```suggestion
      lv_label_set_text_fmt(batteryValue, "%lu%%", batteryPercent);
```